docs/user_guide.md

```diff
@@ -489,8 +489,21 @@ checker configuration:
                         WISELY AND AT YOUR OWN RISK!
 ~~~~~~~~~~~~~~~~~~~~~
 
-Both `--enable` and `--disable` take individual checkers or checker groups as
-their argument and there can be any number of such flags specified.
+Both `--enable` and `--disable` take individual checkers, checker groups or
+checker profiles as their argument and there can be any number of such flags
+specified.
+
+Checker profiles describe custom sets of enabled checks which can be specified
```

Comment:
I think we could highlight this section more in the user guide.
`#### Checker profiles`